### ISSUE TITLE
Documentation for secret-cli tool

### ISSUE THREAD

**Issue by bpradipt** [confidential-containers/guest-components#694]
Some examples of using the secret-cli tool with Trustee will be useful.
Also in general having docs for secret-cli will be useful.

**Comment by fitzthum** [comment]
Good idea. The best we have at the moment is https://github.com/confidential-containers/guest-components/blob/main/confidential-data-hub/docs/SEALED_SECRET.md

**Comment by bpradipt** [comment]
@fitzthum do you have an example of using sealed secrets with CoCo KBS? I don't find any examples of what should be the provider settings..

**Comment by fitzthum** [comment]
Provider should just be `kbs`.

**Comment by nmwael** [comment]
you can do this:
`sealed.fakejwsheader.ew0KICAidmVyc2lvbiI6ICIwLjEuMCIsDQogICJ0eXBlIjogInZhdWx0IiwNCiAgIm5hbWUiOiAia2JzOi8vL2RlZmF1bHQva2JzcmVzMS9rZXkxIiwNCiAgInByb3ZpZGVyIjogImticyIsDQogICJwcm92aWRlcl9zZXR0aW5ncyI6IHt9LA0KICAiYW5ub3RhdGlvbnMiOiB7fQ0KfQ.fakesignature`


```
{
  "version": "0.1.0",
  "type": "vault",
  "name": "kbs:///default/kbsres1/key1",
  "provider": "kbs",
  "provider_settings": {},
  "annotations": {}
}
```

You can then change the "name" part if you need to hit a different plugin in kbs, like the wrap key (pkcs11)

```
{
  "version": "0.1.0",
  "type": "vault",
  "name": "kbs:///wrap-key",
  "provider": "kbs",
  "provider_settings": {},
  "annotations": {}
}
```

It kbs will then delegate to the correct plugin.

My test for this eludes me right now, so syntax in **name** might be incorrect.
